content/en/docs/concepts/configuration/manage-resources-containers.md

```diff
@@ -116,11 +116,11 @@ CPU is always requested as an absolute quantity, never as a relative quantity;
 
 Limits and requests for `memory` are measured in bytes. You can express memory as
 a plain integer or as a fixed-point number using one of these suffixes:
-E, P, T, G, M, k. You can also use the power-of-two equivalents: Ei, Pi, Ti, Gi,
+E, P, T, G, M, k, m (millis). You can also use the power-of-two equivalents: Ei, Pi, Ti, Gi,
```

Comment:
```suggestion
E, P, T, G, M, k, m. You can also use the power-of-two equivalents: Ei, Pi, Ti, Gi,
```

I don't think you need to define m as milli. Especially since none of the others are defined.

Reply:
+1 to this comment

Reply:
I added that because the others are pretty intuitive. E, P, T, G, M, k are very common units for memory size. However, for milli-bytes, I haven't seen any use of this other than here. So if we omit the comment, I'm pretty sure that people just assumes we support both cases for mega.

TBH, I think the best way is just stop supporting this unit for memory size but for some reason, we are allowing it. So I'm trying to warn users about this.

Reply:
You can clarify that you can use one of the following SI unit prefixes (link to https://physics.nist.gov/cuu/Units/prefixes.html) maybe? lowercase m being milli seems self explanatory IMO

Reply:
@shannonxtreme AFAIK, we only support `m` but `c`, `n'.  Technically, yes. `m` for milli is a common self explanatory way. But practically, no one use milli with bytes. So we need to be a little more specific.